packages/syntax-core/src/RichSelect/RichSelectList.tsx

```diff
@@ -180,6 +180,7 @@ function RichSelectList(props: RichSelectListProps): ReactElement {
           [styles.opacityOverlay]: disabled,
           [styles.selectContainerCambio]: themeName === "cambio",
         })}
+        onClick={onClick}
```

Comment:
@jliotta this adds an inaccessible `onClick` to `div` - it's never reachable by keyboard. Could we come up with a better solution?